### DISCUSSION TITLE
I just want to do a crossfade!

### DISCUSSION THREAD

**Discussion by HunterZ** [savonet/liquidsoap#2747] [User Support]
Looking at the documentation for 1.4.3 (the version shipped by Debian), and I can't make heads or tails of what it's talking about for crossfades: https://www.liquidsoap.info/doc-1.4.3/cookbook.html

Also, the "customize your transitions" link here is a 404: https://www.liquidsoap.info/

I have a playlist outputting to icecast. I just want to make it crossfade the songs. The documentation makes zero sense, and I'm a software engineer. This is incredibly frustrating!

**Comment by HunterZ** [comment]
Okay, I figured something out. The cookbook is _really_ confusing, throwing stuff at you about transitions and such, and not starting with a simple example - but the quick start guide gives one, and I didn't think to go back and check until much later.

**Comment by toots** [comment]
Hi @HunterZ . Thanks for your interest into the project and sorry for your frustration. Have you tried reading the liquidsoap book: https://www.liquidsoap.info/doc-2.1.2/book.html ? It does a gradual introduction to all the notions and tools and might be less confusing for a first approach.

Also, keep in mind that `1.4.3` is a pretty old, unmaintained version. We do provide `debian` and `ubuntu` packages for most recent distributions in our install page: https://github.com/savonet/liquidsoap/releases/tag/v2.1.2
